### PR TITLE
add lizmap version to proxy request

### PR DESCRIPTION
**Pull request by nworr**
Proxy Request currently use "Lizmap" as user-agent in request header, that can be usefull for OGC service

the PR add the current lizmap Version

Fix : #2044

Funded by 3liz

### CODE REVIEW

**Comment by rldhont** [comment]
@nworr I think you have to rebase your branch to avoid conflict with phptan tests fixes.

File: lizmap/modules/lizmap/classes/lizmapServices.class.php [hunk 1]
```diff
@@ -282,14 +282,15 @@ public function __construct($readConfigPath, $globalConfig, $ldapEnabled, $varPa
             $this->allowUserAccountRequests = false;
         }
 
-        // set it for external requests, needed for file_get_contents
-        $userAgent = 'Lizmap';
+        $appInfos = \Jelix\Core\Infos\AppInfos::load();
```
Comment (laurentj) [inline lizmap/modules/lizmap/classes/lizmapServices.class.php:285]: Retrieve the userAgent in an `else` statement of the statement ` if (isset($readConfigPath['services']['userAgent'])) {`. 
`AppInfos` loads xml file, and so it takes some resources. We could avoid to load this file if we don't need it.

**Review by laurentj — CHANGES_REQUESTED** [review]

**Comment by rldhont** [comment]
@laurentj requested  changes done.

**Review by rldhont — APPROVED** [review]

**Review by laurentj — CHANGES_REQUESTED** [review]

**Comment by rldhont** [comment]
@nworr it is ready for merging?

**Pull request merged**